app/connector/fhir/src/main/java/io/syndesis/connector/fhir/FhirReadCustomizer.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.fhir;
+
+import java.util.Map;
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.context.FhirVersionEnum;
+import ca.uhn.fhir.rest.api.SummaryEnum;
+import io.syndesis.integration.component.proxy.ComponentProxyComponent;
+import io.syndesis.integration.component.proxy.ComponentProxyCustomizer;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.component.fhir.api.ExtraParameters;
+import org.apache.camel.component.fhir.internal.FhirApiCollection;
+import org.apache.camel.component.fhir.internal.FhirReadApiMethod;
+import org.apache.camel.util.ObjectHelper;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FhirReadCustomizer implements ComponentProxyCustomizer {
+
+    private static final Logger LOG = LoggerFactory.getLogger(FhirReadCustomizer.class);
+
+
+    private String id;
+    private String resourceType;
+    private String version;
+    private Boolean summary;
+    private FhirContext fhirContext;
+
+    @Override
+    public void customize(ComponentProxyComponent component, Map<String, Object> options) {
+        for (Map.Entry<String, Object> stringObjectEntry : options.entrySet()) {
+            LOG.info("setApiMethod key:" + stringObjectEntry.getKey() + "setApiMethod value" + stringObjectEntry.getValue());
```

Comment:
May be better not to log options as info as they may contain sensitive info, maybe not in this case but as general
camel rule this should be avoided.